external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -1786,8 +1787,8 @@ abstract class AvroSuite
   }
 
   // It generates input files for the test below:
-  // "SPARK-31183: compatibility with Spark 2.4 in reading dates/timestamps"
-  ignore("SPARK-31855: generate test files for checking compatibility with Spark 2.4") {
+  // "SPARK-31183, SPARK-37705: compatibility with Spark 2.4/3.2 in reading dates/timestamps"
+  ignore("SPARK-31855: generate test files for checking compatibility with Spark 2.4/3.2") {
```

Comment:
how do we use this test to generate the new testing data files?

Reply:
Just replace `ignore` -> `test`, and run it. For example, I checkout `branch-3.2` and ran the test to generate the golden files. After that, I copied new files to PR's branch.

Reply:
hmm, looking at the code, seems the version is fixed as `2_4_6`

Reply:
Yep, made it more flexible by using `SPARK_VERSION_SHORT`